### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by christophermackey**

### CODE REVIEW

**Review by WalkerTexas-coder — CHANGES_REQUESTED** [review]
Looks good! I think you're missing Test coverage for the home component-- just like the NotFound.test.js Go ahead and make the Home.test.js file and tests for it

File: src/pages/__tests__/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Home has an image', ()=> {
+    let home 
+    beforeEach(()=>{
+        home = shallow(<Home/>)
+    })
+    it('Home renders content', ()=> {
+        // console.log("rendered ")
```
Comment (WalkerTexas-coder) [inline src/pages/__tests__/Home.test.js:14]: Can you delete this console log as it looks like its unused?

File: src/pages/__tests__/NotFound.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('NotFound has an image', ()=> {
+    let notFound 
+    beforeEach(()=>{
+        notFound = shallow(<NotFound/>)
+    })
+    it('NotFound renders content', ()=> {
+        // console.log("rendered ")
```
Comment (WalkerTexas-coder) [inline src/pages/__tests__/NotFound.test.js:14]: Can you delete this console log as it looks like its unused?

File: src/components/__tests__/Header.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import Header from '../Header'
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Header has an image', ()=> {
+    let header 
+    beforeEach(()=>{
+        header = shallow(<Header/>)
+    })
+    it('Header renders content', ()=> {
+        // console.log("rendered ")
```
Comment (WalkerTexas-coder) [inline src/components/__tests__/Header.test.js:13]: Can you delete this console log as it looks like its unused?

File: src/components/__tests__/Footer.test.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import Footer from '../Footer'
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Footer has an image', ()=> {
+    let footer 
+    beforeEach(()=>{
+        footer = shallow(<Footer/>)
+    })
+    it('Footer renders content', ()=> {
+        // console.log("rendered ")
```
Comment (WalkerTexas-coder) [inline src/components/__tests__/Footer.test.js:13]: Can you delete this console log as it looks like its unused?

File: src/App.css [hunk 5]
```diff
@@ -3,20 +3,42 @@ h1 {
   color:#710B79;
 }
 
+/* .body{
+  min-height: 100vh;
+  padding-bottom: 50px;
+} */
+
```
Comment (WalkerTexas-coder) [inline src/App.css:10]: Did you want to keep these styles for later?

**Review by sjproctor — APPROVED** [review]
Looks good! Just be sure to remove any console logs that are no longer necessary.

**Review by WalkerTexas-coder — APPROVED** [review]

**Pull request merged**